src/components/Todos/model.js

```diff
@@ -27,6 +30,10 @@ function searchTodoIndex(todosList, todoid) {
   return null;
 }
 
+// MAKE MODIFICATION STREAM
+// A function that takes the intent on the todo list
+// and returns a stream of functions that expect todosData (the data model)
+// and return a modified version of the data.
```

Comment:
This comment could explain what those functions are. They are "updater" functions. Sort of like reducers in Redux, they take old state and return new state.